src/OSPSuite.UI/Views/Importer/ImporterView.cs

```diff
@@ -29,7 +29,18 @@ public ImporterView()
 
       public void ShowErrorMessage(string message)
       {
-         XtraMessageBox.Show(this, message, "", MessageBoxButtons.OK, MessageBoxIcon.Error);
+         var args = new XtraMessageBoxArgs
+         {
+            Caption = "",
```

Comment:
Why is the caption empty? This should never be the case for a message, especially an error.
At any rate, this code should go

Reply:
Was like this before and just left it there, since I also agree it should go